privaterelay/management/commands/invite_monitor_waitlist.py

```diff
@@ -1,30 +1,109 @@
 from datetime import datetime
+from email.utils import parseaddr
+import logging
 
+from socketlabs.injectionapi.message.basicmessage import BasicMessage
+from socketlabs.injectionapi.message.emailaddress import EmailAddress
+
+from django.conf import settings
+from django.core.exceptions import EmptyResultSet
 from django.core.management.base import BaseCommand
+from django.template.loader import render_to_string
 
+from emails.utils import get_socketlabs_client, socketlabs_send
 from ...models import Invitations, MonitorSubscriber
 
 
+logger = logging.getLogger('events')
+
+
+def email_invited_user(invitee, invitation):
+    context = {
+        'email': invitee.primary_email,
+        'current_domain': settings.SITE_ORIGIN,
+    }
+
+    # Send email invite
+    sl_message = BasicMessage()
+    sl_message.subject = (
+        "You're Invited to the Private Relay Beta!"
+    )
+
+    sl_message.html_body = render_to_string(
+        'emails/beta_invite_html_email.html',
+        context,
+    )
+    sl_message.plain_text_body = render_to_string(
+        'emails/beta_invite_text_email.txt',
+        context,
+    )
+
+    relay_display_name, relay_from_address = parseaddr(
+        settings.RELAY_FROM_ADDRESS
+    )
+    sl_message.from_email_address = EmailAddress(
+        relay_from_address
+    )
+    sl_message.to_email_address.append(
+        EmailAddress(invitee.primary_email)
+    )
+
+    sl_client = get_socketlabs_client()
+    response = socketlabs_send(sl_client, sl_message)
+
+    if not response.result.name == 'Success':
```

Comment:
Not a blocking change but:
```suggestion
    if not response.status_code == 200:
```
`response.result.name` throws a `AttributeError: 'HttpResponse' object has no attribute 'to_json'` exception when it fails to send the email. Recommendation is to check for status_code instead. Need to check to see if the `status_code` is 200 or 201 though.